vmdb/spec/helpers/application_helper/toolbar_builder_spec.rb

```diff
@@ -0,0 +1,2760 @@
+require "spec_helper"
+
+describe ApplicationHelper do
+  before do
+    controller.send(:extend, ApplicationHelper)
+    self.class.send(:include, ApplicationHelper)
+  end
+
+  def method_missing(sym, *args)
```

Comment:
It's scoped to the containing `describe` block (which is actually a class, way under the covers).

It allows the below specs to continue working (and looking) just as they did before.

Specifically, it: 1) makes the methods accessible without needing an explicit receiver, and 2) bypasses the fact that they're now all private.